source/api/presentation/3.0/index.md

```diff
@@ -715,12 +702,12 @@ The value _MUST_ be an array of JSON objects.  Each item _MUST_ have the `id` an
 ```
 
 ##### start
-A link from this Manifest, Sequence or Range to a Canvas, or part of a Canvas, that is contained within it. The reference to part of a Canvas is handled in the same way that Ranges reference parts of Canvases, by adding a fragment to the end of the Canvas's URI specifing the spatial and/or temporal segment of interest.  When processing this relationship, a client _SHOULD_ advance to the specified Canvas, or specified segment of the Canvas, when beginning navigation through the Sequence or Range.  This allows the client to begin with the first Canvas that contains interesting content rather than requiring the user to manually skip past uninteresting content.  The Canvas _MUST_ be included in the first Sequence embedded within the Manifest.
+A link from this Manifest or Range to a Canvas, or part of a Canvas, that is contained within it. The reference to part of a Canvas is handled in the same way that Ranges reference parts of Canvases, by adding a fragment to the end of the Canvas's URI specifing the spatial and/or temporal segment of interest.  When processing this relationship, a client _SHOULD_ advance to the specified Canvas, or specified segment of the Canvas, when beginning navigation through the Range.  This allows the client to begin with the first Canvas that contains interesting content rather than requiring the user to manually skip past uninteresting content.
```

Comment:
> or part of a Canvas, that is contained within it.

or part of a Canvas that is contained within it.

Reply:
OK, I get it. It's still awkward, but what about:

A link from this Manifest or Range, to a Canvas, or part of a Canvas, that is contained within it

Reply:
![image](https://user-images.githubusercontent.com/999701/36597049-c624b176-189f-11e8-8462-f56532f15cbd.png)